spec_linux.go

```diff
@@ -127,6 +127,13 @@ type CPU struct {
 	Mems string `json:"mems"`
 }
 
+type Pids struct {
+	// Maximum number of PIDs. A value <= 0 implies "no limit".
+	// While technically 0 means "no tasks allowed in the cgroup", this does
+	// not make sense in the context of containers.
```

Comment:
Actually, I'm not sure that you're right. In order for the PID limit to make sense, you have to have a process in the cgroup. Nothing stops you from attaching more processes to a cgroup, so it's illogical to claim that a PID limit of 0 has any *real* use. Combine that with the annoyance of not being able to do `new(spec.Linux)` or `spec.Linux{}`, and it's clear that a limit of 0 is not useful.